Engine/src/Engine/Scene.cpp

```diff
@@ -25,12 +25,6 @@ namespace Engine
 #pragma region Runtime Functions
 	void Scene::onRuntimeStart()
 	{
-		if (!initializeGL()) return;
```

Comment:
So onRuntimeStart will assume GL is always initialized? Just want to make sure this check isn't needed anymore, as I'm not too sure why we would want to remove this